src/audio-worklet/basic/hello-audio-worklet/main.js

```diff
@@ -3,24 +3,54 @@
 // found in the LICENSE file.
 
 const audioContext = new AudioContext();
+let oscillator; 
```

Comment:
To make it clear, how about `oscillatorNode`?

Reply:
Thank you for the suggestion. I've updated the variable name to oscillatorNode for better clarity. Please let me know if you have any further suggestions.